ui/src/builder/BuilderCopyText.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+	<span class="tooltip">
+		<span class="tooltiptext">{{ copied }}</span>
+		<span class="copyText" @click="copyText">
+			<slot></slot>
+		</span>
+	</span>
+</template>
+
+<script setup lang="ts">
+import { ref } from "vue";
+const copied = ref("Copy to clipboard");
+
+const copyText = (e) => {
+	const copyText = e.target.textContent;
+	navigator.clipboard.writeText(copyText);
+	copied.value = "Copied!";
+	setTimeout(() => {
+		copied.value = "Copy to clipboard";
+	}, 1000);
+};
+</script>
+
+<style scoped>
+.tooltip {
+	position: relative;
+	display: inline-block;
+}
+
+.tooltip .tooltiptext {
+	visibility: hidden;
+	background-color: var(--builderActionOngoingColor);
+	color: #fff;
```

Comment:
Please use `--builderBackgroundColor`, looks better and if one day we get dark mode or do a Barbie edition we're set.